platform/darwin/docs/guides/Predicates and Expressions.md

```diff
@@ -10,7 +10,9 @@ This document discusses the specific subset of the predicate and expression
 syntax supported by this SDK. For a more general introduction to predicates and
 expressions, consult the
 _[Predicate Programming Guide](https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/Predicates/AdditionalChapters/Introduction.html)_
-in Apple developer documentation.
+in Apple developer documentation. For additional detail on how this SDK has
+extended the `NSExpression` class, consult the [`NSExpression(MGLAdditions)`](./Categories/NSexpression(MGLAdditions).html)
```

Comment:
jazzy might be able to autolink `NSExpression(MGLAdditions)` without having to specify a path to the HTML file.

Reply:
@1ec5 I had thought so too, but it wasn't autolinking properly when I built the docs locally. Hence this kludgy workaround.